worker/contrib/windows/buildbot_service.py

```diff
@@ -104,7 +104,7 @@
 class BBService(win32serviceutil.ServiceFramework):
     _svc_name_ = 'BuildBot'
     _svc_display_name_ = _svc_name_
-    _svc_description_ = 'Manages local buildbot slaves and masters - ' \
+    _svc_description_ = 'Manages local buildbot workers and masters - ' \
                         'see http://buildbot.sourceforge.net'
```

Comment:
Yes, I noticed this too :)
This is on my TODO list (this is not the only place where SourceForge is in codebase), lets fix this later.